### ISSUE TITLE
Latest Glamourer, Penumbra, and Mare updates broke this.

### ISSUE THREAD

**Issue by Petra105** [eqbot/XIVSnapper#20]
Latest Glamourer, Penumbra, and Mare updates broke saving snapshots.

**Comment by Presea-diamond** [comment]
Yep! I'm guessing they changed how those plugins works and it only creates an empty file now.

**Comment by Razann94** [comment]
Same here, it only creates 1kb file for pmp and nothing in the folder when just exporting other than snapshot file

**Comment by RyzenStellar** [comment]
Same here, it only creates 1kb file for pmp and nothing in the folder when just exporting other than snapshot file

**Comment by awoogawoooga** [comment]
Yeah same, doesnt load anything, saves 1kb and doesnt export character so i cant load on brio or export as pmp

**Comment by orenji** [comment]
Same for me as well.
